researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/task/tracked/TrackedDataSelectedItemsInterface.java

```diff
@@ -0,0 +1,17 @@
+package org.sagebionetworks.bridge.researchstack.task.tracked;
+
+import org.researchstack.backbone.result.StepResult;
+
+import java.util.List;
+
+/**
+ * Created by TheMDP on 3/23/17.
+ */
+
+public interface TrackedDataSelectedItemsInterface {
```

Comment:
Sounds good, feel free to merge whenever you're ready. I've already approved this PR since everything looks greatand makes sense as is